src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -223,6 +227,10 @@ trait WorkflowComponent {
       filter(_.id === id)
     }
 
+    def findWorkflowByExternalIdAndSubmissionId(externalId: String, submissionId: UUID): WorkflowQueryType = {
```

Comment:
I see in the schema that the workflow id is called external_id (not sure why), but it seems like this function and getByExternalId would be clearer to read if the word External was replaced with Workflow